src/bindings/python/tests/test_graph/test_preprocess.py

```diff
@@ -594,7 +594,7 @@ def test_ngraph_set_layout_by_str_thow_exception(layout):
 
     with pytest.raises(RuntimeError) as e:
         ppp.input().model().set_layout(layout)
-    assert "Layout name is invalid" in str(e)
+        assert "Layout name is invalid" in str(e)
```

Comment:
Why there is an additional indentation? I do not recall it from pytest docs. There is always assert outside the context manager.

Reply:
discussed offline